flowmachine/Pipfile

```diff
@@ -32,8 +32,8 @@ pytest-asyncio = "*"
 pytest-cov = "*"
 pytest-testmon = "*"
 asynctest = "*"
-flowmachine = {editable = true,path = "."}
```

Comment:
Why's this removed?

Reply:
Because otherwise we can't run `pipenv install --dev` (which implicitly installs flowmachine in case this line is present) before `COPY . /app/` in the Dockerfile. However, we want to run `pipenv install --dev` beforehand to take advantage of docker layer caching (so that we don't need to reinstall all dependencies if the source code changes).

Reply:
(Otherwise I would have preferred to leave it in but I didn't see a way of having both.)